nebula/ui/components/MZSystemAlert.qml

```diff
@@ -72,13 +72,7 @@ MZAlert {
                     target: alertBox
                     //% "Background service error"
                     alertText: qsTrId("vpn.alert.backendServiceError")
-                    //% "Restore"
-                    //: Restore a service in case of error.
-                    alertActionText: qsTrId("vpn.alert.restore")
                     visible: true
-                    onActionPressed: ()=>{
-                        VPN.backendServiceRestore();
```

Comment:
Can we remove `backendServiceRestore()` from the code altogether since it's not doing anything?

```c++
void MozillaVPN::backendServiceRestore() {
  logger.debug() << "Background service restore request";
  // TODO
}
```

Reply:
This method is still callable through a few other code paths in the project, so I didn't feel it was quite ready for removal until we can clean the rest of them up. In particular, we can still get here from an `ErrorHandler::RemoteServiceError` or an `ErrorHandler::UnrecoverableError`

Although, the name no longer makes sense for these cases.

Reply:
I have opted to remove the method.